backend/nokia-innovative-project/src/main/java/com/nokia/library/nokiainnovativeproject/config/SecurityConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.nokia.library.nokiainnovativeproject.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.userdetails.User;
+
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig  extends WebSecurityConfigurerAdapter {
```

Comment:
Why security conf is added in entities feature? 
I think that you do not need it here. This configuration can be done in separate feature.